raulin0/backup-juggler
# release: releases version 0.3.0 to the master branch

## Description
This pull request adds issue templates to improve the issue creation process. It includes the following changes:

- Created a bug report template to help users report bugs effectively.
- Created a feature request template to facilitate the submission of new feature ideas.
- Created a documentation issue template to streamline the reporting of documentation-related issues.

## Changes Made
- Added a bug report template (bug_report.md) to provide a structured format for reporting bugs. The template includes sections for issue description, steps to reproduce, expected behavior, actual behavior, and additional information.
- Added a feature request template (feature_request.md) to guide users in submitting feature requests. The template includes sections for feature description, use case, and additional information.
- Added a documentation issue template (documentation_issue.md) to assist users in reporting documentation-related issues. The template includes sections for issue description, suggested improvement, and additional information.

## Motivation and Context
The addition of issue templates improves the issue creation process by standardizing the information required from users. This helps in effectively capturing bug reports, feature requests, and documentation issues. By providing clear guidelines and a structured format, it enhances communication between users and the development team, leading to better issue resolution and overall project management.

## How to Test
1. Create a new issue using the bug report template and ensure that all relevant sections are filled out accurately.
2. Create a new issue using the feature request template and provide a comprehensive description of the desired feature.
3. Create a new issue using the documentation issue template and specify the issue with the existing documentation along with suggested improvements.
4. Verify that the issue templates provide clear instructions and guidance to users.
5. Check that the submitted issues include all the necessary information according to the respective templates.

## Related Issue(s)
N/A

## Checklist
- [x] Issue templates have been added.
- [x] Templates include relevant sections for capturing necessary information.
- [x] Templates provide clear instructions and guidelines to users.

Please review and merge this pull request at your earliest convenience. Your feedback and suggestions for improvement are highly appreciated. Thank you!
